packages/twenty-server/src/workspace/workspace-sync-metadata/core-objects/user.object-metadata.ts

```diff
@@ -0,0 +1,33 @@
+import { FieldMetadataType } from 'src/metadata/field-metadata/field-metadata.entity';
+import { RelationMetadataType } from 'src/metadata/relation-metadata/relation-metadata.entity';
+import { FieldMetadata } from 'src/workspace/workspace-sync-metadata/decorators/field-metadata.decorator';
+import { Gate } from 'src/workspace/workspace-sync-metadata/decorators/gate.decorator';
+import { IsSystem } from 'src/workspace/workspace-sync-metadata/decorators/is-system.decorator';
+import { ObjectMetadata } from 'src/workspace/workspace-sync-metadata/decorators/object-metadata.decorator';
+import { RelationMetadata } from 'src/workspace/workspace-sync-metadata/decorators/relation-metadata.decorator';
+import { BaseObjectMetadata } from 'src/workspace/workspace-sync-metadata/standard-objects/base.object-metadata';
+import { WorkspaceMemberObjectMetadata } from 'src/workspace/workspace-sync-metadata/standard-objects/workspace-member.object-metadata';
+
+@ObjectMetadata({
+  namePlural: 'users',
+  labelSingular: 'User',
+  labelPlural: 'Users',
+  description: 'A User',
+  dataSourceSchema: 'core',
+})
+@Gate({
+  featureFlag: 'IS_STANDARD_CORE_RELATIONSHIPS_ENABLED',
+})
+@IsSystem()
+export class UserObjectMetadata extends BaseObjectMetadata {
+  @FieldMetadata({
+    type: FieldMetadataType.RELATION,
+    label: 'Workspace Member',
+    description: 'Related workspace member',
+  })
+  @RelationMetadata({
+    type: RelationMetadataType.ONE_TO_ONE,
```

Comment:
It's a tricky ONE_TO_ONE because in practice we could have a User in multiple workspace so a user could point toward multiple workspace members, each one identifying the same user in different workspaces. It should work because the joinColumn is inside the workspace schema but it should actually be a ONE_TO_MANY but toward different schemas 🤔 
cc @charlesBochet @magrinj